tests/test.rfb.js

```diff
@@ -3508,6 +3508,9 @@ describe('Remote Frame Buffer protocol client', function () {
             // client coordinate calculations
             client.focusOnClick = false;
 
+            // We need to keep track of MouseEvent.buttons state
+            client._buttonsState = 0;
```

Comment:
This feels like it obfuscating a lot of what the tests are supposed to do. Can't we make sure the tests provide this?